abodepy/event_controller.py

```diff
@@ -67,12 +67,37 @@ def add_device_callback(self, devices, callback):
                 raise AbodeException((ERROR.EVENT_DEVICE_INVALID))
 
             _LOGGER.debug(
-                "Subscribing to updated for device_id: %s", device_id)
+                "Subscribing to updates for device_id: %s", device_id)
 
             self._device_callbacks[device_id].append((callback))
 
         return True
 
+    def remove_device_callback(self, devices):
+        """Unregister a device callback."""
+        if not devices:
+            return False
+
+        if not isinstance(devices, (tuple, list)):
+            devices = [devices]
+
+        for device in devices:
+            device_id = device
```

Comment:
Not sure if this is necessary since we're only passing in a single device. Perhaps this implementation allows more flexibility though for other uses outside of Home Assistant?